app/src/main/java/com/duckduckgo/app/browser/useragent/UserAgentProvider.kt

```diff
@@ -111,6 +127,7 @@ class UserAgentProvider constructor(@Named("defaultUserAgent") private val defau
         const val SPACE = " "
         val mobilePrefix = "Mozilla/5.0 (Linux; Android ${Build.VERSION.RELEASE})"
         val desktopPrefix = "Mozilla/5.0 (X11; Linux ${System.getProperty("os.arch")})"
+        val fallbackDefaultUA = "$mobilePrefix AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/96.0.4664.104 Mobile Safari/537.36"
```

Comment:
I'm using the latest chrome version here as part of our UA. should not be an issue. If anyone knows a better way to handle this, please share.